.github/workflows/docker-unified.yml

```diff
@@ -29,8 +29,14 @@ env:
   DATAHUB_MCE_CONSUMER_IMAGE: "acryldata/datahub-mce-consumer"
   DATAHUB_KAFKA_SETUP_IMAGE: "acryldata/datahub-kafka-setup"
   DATAHUB_ELASTIC_SETUP_IMAGE: "acryldata/datahub-elasticsearch-setup"
-  DATAHUB_MYSQL_SETUP_IMAGE: "acryldata/datahub-mysql-setup"
-  DATAHUB_UPGRADE_IMAGE: "acryldata/datahub-upgrade"  
+  #### IMPORTANT ####
+  #### THIS IS A CHANGE TO PREVENT OSS QUICKSTART INSTABILITY, DO NOT OVERWRITE THIS CHANGE IN MERGES ####
+  #### IMPORTANT ####
+  DATAHUB_MYSQL_SETUP_IMAGE: "acryldata/acryl-datahub-mysql-setup"
+  DATAHUB_UPGRADE_IMAGE: "acryldata/acryl-datahub-upgrade"
```

Comment:
So this is not a complete fix for this repo. Yes, it will fix OSS and therefore quickstart stability once again. However any test that relies on these images, i.e. smoke tests will fail with the reverse problem. It'll be using OSS built `datahub-upgrade` with a gms build from this mirror. Any testing in this repo would need to make sure to use these images instead of the OSS ones which are used by default.

Reply:
This image id is the one that gets passed through to all the docker-unified subtasks though I thought, so it would use this built image in the tests for this repo. I'll verify.

Reply:
https://github.com/acryldata/datahub/blob/7c37dae47bd623923647af5b719d66473c888e8c/.github/workflows/docker-unified.yml#L426-L501

Reply:
Smoke tests are fine because they don't use head, the problem is that when quickstart uses head, sometimes a dot release version from releasing the python libs from this repo gets published to the head tag and doesn't align. The smoke test for that would be using an image with the same version tag (different tag name though which is fine).

Reply:
@anshbansal Build is currently red from your changes to the workflows, going to go ahead and merge this in to alleviate the instability issues with quickstart.